src/main/java/com/aws/greengrass/shadowmanager/PubSubIntegrator.java

```diff
@@ -111,6 +113,13 @@ ShadowRequest extractShadowFromTopic(String topic) {
      */
     private void handlePublishedMessage(PublishEvent publishEvent) {
         String topic = publishEvent.getTopic();
+
+        if (isResponseMessage(topic)) {
+            logger.atDebug().kv(LOG_TOPIC, topic)
+                    .log("Not processing message since it is a response message to a shadow operation");
+            return;
```

Comment:
Will it work that we don't subscribe to those response topics at all but only subscribe to the ones we need to react to?

Reply:
We had a broader discussion around this today. We might need some work in PubSub.

Reply:
> Will it work that we don't subscribe to those response topics at all but only subscribe to the ones we need to react to?

+1